dotcom-rendering/src/components/Ad.amp.tsx

```diff
@@ -111,7 +111,7 @@ export const Ad = ({
 			// corresponding primary size.
 			data-multi-size-validation="false"
 			data-npa-on-unknown-consent={true}
-			data-loading-strategy="prefer-viewability-over-views"
+			data-loading-strategy="0.5"
```

Comment:
@emma-imber Is this OK as a string? I would have thought this would have to be a float: https://amp.dev/documentation/components/amp-ad#data-loading-strategy-(optional)

Reply:
It's a string according to the docs here: https://github.com/ampproject/amphtml/blob/main/extensions/amp-ad-network-doubleclick-impl/lazy-fetch.md